script/default-site

```diff
@@ -12,7 +12,7 @@ workdir=$(pwd)
 echo "$0: creating new default site"
 if [[ "$(uname -s)" != "Linux" ]]
 then 
-  exec exe/jekyll new tmp/default-site
+   `exe/jekyll new tmp/default-site`
```

Comment:
I added the if..fi block there..
The revert has been pushed.. and it passed Appveyor..